web/core/src/jsMain/kotlin/androidx/compose/web/elements/Elements.kt

```diff
@@ -249,7 +249,7 @@ fun Select(
 )
 
 @Composable
-fun DOMScope<HTMLUListElement>.Option(
+fun DOMScope<HTMLSelectElement>.Option(
```

Comment:
I think `DomScope<...>` can be omitted here at all, since `<option>` can be use not only in `<select>`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option

let's keep it as `fun Option(...)` without any scope.